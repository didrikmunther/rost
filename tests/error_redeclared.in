let a = 123;
let a = 123;