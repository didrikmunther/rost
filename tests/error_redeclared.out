  --> ./tests/error_redeclared.in:2:5, ./tests/error_redeclared.in:1:5 => CompilerError
1 | let a = 123
  |     ^
  |     └─ Already declared here
2 | let a = 123
  |     ^
  |     └─ Redeclared variable: a